bpython/curtsiesfrontend/manual_readline.py

```diff
@@ -212,7 +212,7 @@ def back_word(cursor_offset, line):
     return (last_word_pos(line[:cursor_offset]), line)
 
 
-@edit_keys.on('<PADDELETE>')
+@edit_keys.on('<DELETE>')
```

Comment:
On Manjaro Linux (Arch) `<PADDELETE>` doesn't worked either.

Pypi's package doesn't have this change.